frontend/src/Services/Requests/socials.ts

```diff
@@ -0,0 +1,7 @@
+import { Fetcher } from '../Fetcher';
+
+export class SocialRequests {
+  static getAllSocials = async () => {
+    return Fetcher.get('socials');
```

Comment:
I suggest to use slash before URLs
```suggestion
    return Fetcher.get('/socials');
```